dev-tex/harvard: update HOMEPAGE, #533822

wimmuskee opened the pull request:
Gentoo-Bug: https://bugs.gentoo.org/533822
Gentoo-Bug: https://bugs.gentoo.org/470062

Added a revbump because added RDEPEND (similar to other latex packages). The package would not compile without pdflatex (and thus needs latexextra).

Was not sure whether to remove the regular 2.0.5.ebuild. This one does not build on a fresh stage3 environment.

nvinson commented:
> Gentoo-Bug: https://bugs.gentoo.org/533822
> Gentoo-Bug: https://bugs.gentoo.org/470062

These tags should be part of the actual git commit message.  I would also recommend changing the commit title to "dev-tex/harvard: Update upstream"

> Added a revbump because added RDEPEND (similar to other latex packages). The package would not compile without pdflatex (and thus needs latexextra).

pdflatex is part of the texlive-latex package, not latexextra.  Furthermore, the ebuild inherits the latex-package eclass and that eclass should pull in texlive-latex as a dependency (latex-package.eclass depends on virtual/latex-base. virtual/latex-base depends on texlive-latexrecommended. texlive-latexrecommended depends on texlive-latex).

nvinson commented on dev-tex/harvard/harvard-2.0.5-r1.ebuild:
```diff
@@ -0,0 +1,17 @@
+SRC_URI="mirror://gentoo/${P}.tar.bz2"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 ppc x86"
```
When performing a revbump all keywords have to be reverted to testing.  In this particular case that means
```
KEYWORDS="amd64 ppc x86"
```
should become
```
KEYWORDS="~amd64 ~ppc ~x86"
```

wimmuskee commented:
@nvinson, you are correct, the inherit latex-package indeed pulls the required dependencies (apparently I am testing in a stage3 chroot where this doesn't work)

removed the revbump and rebased

As for the commit message, I've done almost 20 more like it, and never got any complaints. In my opinion "update upstream" is too generic, while "update HOMEPAGE" clearly states what has been changed.

nvinson commented:
> As for the commit message, I've done almost 20 more like it, and never got any complaints. In my 

Upon review, it appears that 10 out of those 29 commits did not adhere to the requirements as stated in the [Gentoo Developers Manual](https://devmanual.gentoo.org/ebuild-maintenance/index.html#git-commit-message-format).  That said, it's the job of both the committer and the author to ensure that the requirements stated in the manual are met.  Mistakes will be made and errors will not always be caught, but those incidents should not be use as justification for why a commit should not be corrected.

The commit message given in this PR also does not match the manual's requirements, so please update the commit message so that it does.  If you feel the manual is out-of-date on this point, please file a bug asking to have the linked section reviewed and corrected.

> opinion "update upstream" is too generic, while "update HOMEPAGE" clearly states what has been changed.

To me "update HOMEPAGE" suggests only the HOMEPAGE variable was updated.  Your commit updates both the HOMEPAGE and SRC_URI variables.  Therefore, while the current message is more specific than "update upstream", it appears to be more inaccurate as well in my opinion.

aballier commented:
merged, thanks!
